### DISCUSSION TITLE
Example integration with front-end

### DISCUSSION THREAD

**Discussion by atlasstrategic** [dinoperovic/django-salesman#27] [Ideas]
There are some great options for front-end deploys using headless back-ends—would be good to see Django Salesman examples there too.

## Example integrations
- [Saleor demo using Deno Merch](https://github.com/saleor/deno-merch)
- [Saleor demo using NextJs Commerce](https://github.com/vercel/commerce)

**Comment by dinoperovic** [comment]
@atlasstrategic yes, that would be nice to have. I will see what I can do.
